dotCMS/src/main/java/com/dotcms/cluster/common/ClusterServerActionThread.java

```diff
@@ -56,6 +63,24 @@ public void run() {
 					}
 					connection.commit();
 				}
+				    
+				if(LicenseUtil.getLevel()==100 ){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 100 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)